developer-docs-site/docs/guides/aptos-apis.md

```diff
@@ -23,6 +23,10 @@ Most integrations into the Aptos blockchain benefit from a holistic and comprehe
 * The state is the representation of all transaction outputs up to a specific version. In other words, a state version is the accumulation of all transactions inclusive of that transaction version.
 * As transactions execute, they may emit events. [Events](../concepts/events.md) are hints about changes in on-chain data.
 
+:::important
+Ensure the [fullnode](../nodes/aptos-deployments.md) you're communicating with has reached the version that your transaction progressed the blockchain to as there can be a lag from the fullnodes retrieving state from [validator fullnodes](../concepts/fullnodes.md), which in turn rely upon [validator nodes](../concepts/validator-nodes.md) as the source of truth. See the [Aptos Explorer](https://explorer.aptoslabs.com/) for each network to search for the transactions.
```

Comment:
This reads funny, I can't quite put my finger on it.

```
::important
[Fullnodes](../nodes/aptos-deployments.md) that you communicate with may be delayed behind the state of the chain from [validator nodes](../concepts/validator-nodes.md).  This is due to state propagation across nodes from commit time.  See the Aptos Explorer for each network to search for the most recent transactions.
```

Technically the most recent transactions may not show up on the explorer for the same reason, but in a UI perspective, it's as up to date as possible.